infrastructure/tari_script/src/op_codes.rs

```diff
@@ -258,6 +258,13 @@ pub enum Opcode {
 }
 
 impl Opcode {
+    pub fn get_version(&self) -> OpcodeVersion {
+        match self {
+            Opcode::CheckMultiSigVerify(..) => OpcodeVersion::V1,
+            _ => OpcodeVersion::V0,
```

Comment:
This should be expanded, so that new opcodes are not accidentally added to an old version
```suggestion
// ... expand existing,
            _ => OpcodeVersion::V2,
```

Reply:
Happy otherwise